### PR TITLE
Set full content-type in all frameworks

### PR DESCRIPTION
**Pull request by mcollina**
I did a bit of analysis of the latest result, and I've noticed a discrepancy in the ratio between the bandwidth and the number of request/second. Turns out that a lot of the frameworks are not setting the the content-type correctly. Fastify did the same in the beginning :).

Anyway, this PR normalizes it or all frameworks that I could spot. We should also add some unit tests for this.

### CODE REVIEW

File: benchmarks/bare.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const server = require('http').createServer(function (req, res) {
-  res.setHeader('content-type', 'application/json')
+  res.setHeader('content-type', 'Content-Type: application/json; charset=utf-8')
```
Comment (delvedor) [inline benchmarks/bare.js]: I think I'm missing something here, why you are adding the header name inside the header value?

Comment (aichholzer) [reply benchmarks/bare.js]: Same question as @delvedor here...

Comment (mcollina) [reply benchmarks/bare.js]: I just did some bad copy-paste. The actual difference is in the `charset` addition, that's the full content-type that we should be measuring against, as that's what understood by all browsers.

**Comment by mcollina** [comment]
@JamesKyburz would you mind updating the results after this?

**Comment by JamesKyburz** [comment]
@mcollina I can update the benchmarks later today.

**Comment by aichholzer** [comment]
See https://github.com/fastify/benchmarks/pull/78

**Pull request merged**